minard/burst.py

```diff
@@ -0,0 +1,52 @@
+import couchdb
+from . import app
+
+def load_burst_runs(offset):
+    """
+    Returns a dictionary with the burst runs loaded from couchdb. The dummy itterator is used as key
+    to keep the ordering from the couchdb query. The content of the couchdb document is stored as values.
+    This loads ALL the documents in burst database, ordered by run_subrun_burst logic.
+    The logic to limit and split the results per page was implemented.
+    """
+    server = couchdb.Server("http://snoplus:"+app.config["COUCHDB_PASSWORD"]+"@"+app.config["COUCHDB_HOSTNAME"])
+    db = server["burst"]
+
+    i = 0 #the counter keeps the ordering correct
+    results = {}
+    limit = 10    #this changes the # of lines per page, can be modified when the DB gets big
+    offset = offset
+    page = (offset / limit)
+    skip = page * limit
```

Comment:
Not sure if I understand this. It looks here like skip will just equal offset? Or is the upper divide supposed to be an integer divide, i.e. `//`?

Reply:
hmm you are right, that is completely unnecessary there (sorry)